docs/network-flow-visibility.md

```diff
@@ -43,6 +44,45 @@ For visualizing the network flows, Antrea monitors the flows in Linux conntrack
 module. These flows are converted to flow records, and then flow records are post-processed
 before they are sent to the configured external flow collector.
 
+## Getting Started
+
+To get started, let's ensure you have installed Antrea with the Flow Exporter
+feature enabled by setting the following config parameter in `antrea.yaml`.
+
+```yaml
+  antrea-agent.conf: |
+    # FeatureGates is a map of feature names to bools that enable or disable experimental features.
+    featureGates:
+    # Enable flowexporter which exports polled conntrack connections as IPFIX flow records from each
+    # agent to a configured collector.
+      FlowExporter: true
+```
+
+To enable both the Grafana Flow Collector and the
+[NetworkPolicy Recommendation](./networkpolicy-recommendation.md) features in
+Theia, please clone the repository, and install Theia and Flow Aggregator by
+runnning the following commands.
+
+```bash
+git clone https://github.com/antrea-io/theia.git
+helm install theia theia/build/charts/theia --set sparkOperator.enable=true -n flow-visibility --create-namespace
+helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true,recordContents.podLabels=true -n flow-aggregator --create-namespace
+```
+
+To enable only the Grafana Flow Collector feature, please clone the repository,
+and install Theia and Flow Aggregator by runnning the following commands.
+
+```bash
+git clone https://github.com/antrea-io/theia.git
+helm install theia theia/build/charts/theia -n flow-visibility --create-namespace
+helm install flow-aggregator theia/build/charts/flow-aggregator --set clickHouse.enable=true -n flow-aggregator --create-namespace
+```
+
+You can refer to
+[Antrea Flow Exporter](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#flow-exporter)
+and [Antrea Flow Aggregator](https://github.com/antrea-io/antrea/blob/main/docs/network-flow-visibility.md#flow-aggregator)
+for more information.
+
 ## Grafana Flow Collector
```

Comment:
I feel we should reorganize the sections. For example:

1. Move contents of "Purpose" and "Grafana & ClickHouse" to "Overview", and also describe "Policy Recommendation" in the "Overview".
2. Add a section about "Grafana Flow Collector Installation and Configuration", and move "Deployment Steps" and "Configuration" there.
3. Add a section about "Grafana Dashboards" for dashboards.

But I am fine to merge the current changes first, and we can revise later.